### ISSUE TITLE
[A11Y] Active page indicator missing in responsive view

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/Orange-Boosted-Bootstrap#2849]
### Prerequisites

- [x] I have [searched](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [x] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [x] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The [Orange Navbar component ](https://boosted.orange.com/docs/5.3/components/orange-navbar/) displays the active item in orange with an underline to emphasize the current state.

![](https://github.com/user-attachments/assets/b3464fc4-b8c3-4211-a0b7-dbadc70ba15d)

The underline is displayed only for a min-width: 1024px (`@media (min-width: 1024px)`, so when we display the page in responsive mode (width 320px), the underline is removed. **From an accessibility POV this is not conform because color shall not be the only way of passing the information.**

![](https://github.com/user-attachments/assets/24a1dafc-e533-4dfa-aa49-f3971623ed0d)

### Reduced test cases

Go to  [Orange Navbar component ](https://boosted.orange.com/docs/5.3/components/orange-navbar/) and activate responsive in devtool with 320px width

![Image](https://github.com/user-attachments/assets/6a240f1d-30d3-414f-bc99-2e7cb770cb67)

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Firefox

### What version of Boosted are you using?

v5.3

**Comment by louismaximepiton** [comment]
Hey, thanks for your contribution,

It's indeed an issue we raised while developing the component but it's design issue right now (see https://design.orange.com/0c1af118d/p/84c686-global-menu/t/page-84c686-57065600-19) I'll raise the issue again to the design team.

**Comment by MaxLardenois** [comment]
Boosted only, won't fix
